clients/drcachesim/common/trace_entry.h

```diff
@@ -467,16 +467,18 @@ marker_type_is_function_marker(const trace_marker_type_t mark)
 // size for all architectures and DR's define is available ifdef X86 only.
 #define MAX_ENCODING_LENGTH 17
 
-// This is the data format generated by the online tracer and produced after
-// post-processing of raw offline traces.
-// The reader_t class transforms this into memref_t before handing to analysis tools.
-// Each trace entry is a <type, size, addr> tuple representing:
-// - a memory reference
-// - an instr fetch
-// - a bundle of instrs
-// - a flush request
-// - a prefetch request
-// - a thread/process
+/**
+ * This is the data format generated by the online tracer and produced after
+ * post-processing of raw offline traces.
+ * The reader_t class transforms this into memref_t before handing to analysis tools.
```

Comment:
`#` for links (or is there an include file issue?)

Reply:
Done. No issue. Just forgot.